openeogeotrellis/backend.py

```diff
@@ -587,9 +587,15 @@ def load_result(self, job_id: str, user_id: str, load_params: LoadParameters, en
 
         return image_collection.filter_bands(band_indices) if band_indices else image_collection
 
-    def load_ml_model(self, job_id: str) -> 'JavaObject':
-        directory = GpsBatchJobs.get_job_output_dir(job_id)
-        filename = "file:" + str(pathlib.Path(directory) / "randomforest.model")
+    def load_ml_model(self, job_id: str, dest_path: Path) -> 'JavaObject':
+        filename = None
+        if job_id is not None:
+            directory = GpsBatchJobs.get_job_output_dir(job_id)
+            filename = "file:" + str(Path(directory) / "randomforest.model")
```

Comment:
how can you be sure that "randomforest.model" is the file name here?

Reply:
This will have to become more generic in the future but currently a batch job can only contain random forest models.

A good solution for later would be to parse the batch job metadata file in the job directory and get the model url from there.